### ISSUE TITLE
5.0.3 - Could not find de.authada.library:aal:4.4.0

### ISSUE THREAD

**Issue by slavikdenis** [idnow/de.idnow.android#80]
```
> Task :app:mergeStageDebugAssets FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeStageDebugAssets'.
> Could not resolve all files for configuration ':app:stageDebugRuntimeClasspath'.
   > Could not find de.authada.library:aal:4.4.0.
     Searched in the following locations:
       - file:/Users/redacted/.m2/repository/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - http://maven.regulaforensics.com/RegulaDocumentReader/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - http://maven.regulaforensics.com/RegulaDocumentReader/Beta/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - https://dl.google.com/dl/android/maven2/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - https://jcenter.bintray.com/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - https://www.jitpack.io/de/authada/library/aal/4.4.0/aal-4.4.0.pom
       - https://raw.githubusercontent.com/idnow/de.idnow.android/master/de/authada/library/aal/4.4.0/aal-4.4.0.pom
     Required by:
         project :app > de.idnow.sdk:idnow-android-sdk:5.0.3 > de.idnow.android.eid:idnow-android-eid-sdk:1.6.0
```

Note: we are not using eID SDK and neither .aar file. We are using the SDK as described [here](https://github.com/idnow/de.idnow.android#how-to-import-the-sdk)

**Comment by acollazomayer** [comment]
Same isssue here @idnow

**Comment by slavikdenis** [comment]
Fixed via version `5.0.5` it seems like. Closing
